action/index.js

```diff
@@ -73,14 +73,16 @@ async function main() {
   }
 
   core.debug(`Discovering assets in "${cwd}"`);
+  const includes = include.flatMap((i) => i.split(","));
+  const excludes = exclude.flatMap((e) => e.split(","));
+  // Exclude node_modules by default unless explicitly specified
+  if (!includes.some((i) => i.includes("node_modules"))) {
+    excludes.push("**/node_modules");
+  }
```

Comment:
Prior to #317, `action/deps.js` was manually modified to exclude node_modules (see [#222](https://github.com/denoland/deployctl/pull/222/files#diff-f9f204a50fc8382442bcf662817c45a184b73283ee9369b98b19369526df0334R4039)). #317 overwrote this manual change and has caused `Always exclude node_modules directory` test case to fail. To fix this, I added node_modules to `exclude` in the same way as https://github.com/denoland/deployctl/blob/acb2ba33df6b146bc8705a1d1c16283d9fef9e78/src/config_inference.ts#L235-L237